utils/gatewayUtils.js

```diff
@@ -20,31 +20,36 @@ const runGqlQuery = async ({ query, variables, jwt }) => {
   });
 };
 
-const uploadFileFromString = (fileData, fileName, gqlVariableName) => {
-  return { fileData, fileName, gqlVariableName };
+const uploadFileFromString = (fileData, fileName) => {
+  return { fileData, fileName };
 };
 
 /**
  *
- * @param options.files is an object of structure {fileData, fileName, gqlVariableName} - use stringToUploadFile() method to generate
+ * @param options.files is an object of structure {fileData, fileName } - use stringToUploadFile() method to generate
  */
-const runGqlUpload = async ({ query, variables, jwt, files }) => {
+const runGqlUpload = async ({ query, variables, jwt, files, asArray = true }) => {
   const formData = new FormData();
   const updatedVariables = { ...variables };
   const map = {};
+  updatedVariables.files = [];
```

Comment:
just to help with readability `const updatedVariables = { ...variables, files:[]  };` is preferred